OpenOversight/app/models/database.py

```diff
@@ -710,8 +584,27 @@ class User(UserMixin, BaseModel):
     is_disabled = db.Column(db.Boolean, default=False)
     dept_pref = db.Column(db.Integer, db.ForeignKey("departments.id"))
     dept_pref_rel = db.relationship("Department", foreign_keys=[dept_pref])
-    classifications = db.relationship("Image", back_populates="user")
-    tags = db.relationship("Face", backref="user")
+
+    # creator backlinks
+    classifications = db.relationship(
+        "Image", back_populates="creator", foreign_keys="[Image.created_by]"
```

Comment:
🔬:
- You could make the `creator` string a consant.
- Are the brackets needed syntax?

Reply:
Added `creator` as a constant and removed the brackets